code/datums/dog_fashion.dm

```diff
@@ -191,6 +191,10 @@
 	..()
 	desc = "That's Definitely Not [M.real_name]."
 
+/datum/dog_fashion/head/cone
+	name = "REAL_NAME of the cone"
```

Comment:
I feel this this leaves something trailing and missing.

`Ian of the cone` what?

Reply:
Would it be better if i just left it as "REAL_NAME" ?

Reply:
Yes